impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js

```diff
@@ -523,6 +524,12 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
     } else {
       domClass.add(node, "filterDialogHidden");
     }
+    if (this.matchComparator.value === pentaho.pda.Column.CONDITION_TYPES.IS_NULL || this.matchComparator.value === pentaho.pda.Column.CONDITION_TYPES.NOT_NULL) {
+      domClass.add(this.parameterContainer, "filterDialogHidden");
+    }
+    else {
```

Comment:
Just above, you have an if/else formatted in a different way... Please be more careful in the future, so that we all don't spend time on these things :-(